includes/handler/class-create.php

```diff
@@ -79,6 +80,9 @@ public static function handle_create( $activity, $user_id, $activity_object = nu
 			$reaction = \get_comment( $state );
 		}
 
+		/* @ticket https://github.com/Automattic/wordpress-activitypub/issues/1001 */
+		add_to_outbox( $activity, 'Announce', $user_id, ACTIVITYPUB_CONTENT_VISIBILITY_PUBLIC );
+
```

Comment:
This makes only sense if you use the `ACTIVITYPUB_ACTOR_AND_BLOG_MODE`.

```suggestion
		/* @ticket https://github.com/Automattic/wordpress-activitypub/issues/1001 */
		if ( ACTIVITYPUB_ACTOR_AND_BLOG_MODE === \get_option( 'activitypub_actor_mode' ) ) {
			add_to_outbox( $activity, 'Announce', $user_id, ACTIVITYPUB_CONTENT_VISIBILITY_PUBLIC );
		}

```

I would also recommend to only boost the Object-ID URL instead of the complete Activity. Not sure though if that is enough information.